### ISSUE TITLE
Line Breakpoints (not debugging)

### ISSUE THREAD

**Issue by Rycochet** [Everlaw/nbts#122]
This is related to #9 slightly, but not meant as a duplicate.

When coding in JS I often add a Line Breakpoint (Click just to the left of the the line number) as it gives an easy way to go back to a point you want to return to (by clicking the red line in the scrollbar).

I don't care about breakpoints in the debugging sense (sourcemaps and multiple browsers do enough), purely being able to mark specific lines I go back to in long functions and files where the Navigator panel isn't helpful.

**Comment by Chris2011** [comment]
Breakpoints are not made for this. What you want are bookmarks and this is also implemented. Right click in front of the line -> bookmarks -> toogle bookmars.

**Comment by Chris2011** [comment]
I think you can add a shortcut for this by your own.

**Comment by Rycochet** [comment]
@Chris2011 Never use them simply because they're not obvious - not least because they don't show up as an obvious colour in the scrollbar (it's almost indistinguishable from the "other instances of the selected name"). At least the red breakpoint looks the same colour as an error - which shouldn't be in there anyway ;-)

**Comment by Chris2011** [comment]
Yeah but this is not what breakpoints are made for, breakpoints are for debugging, everything else are bookmarks. And the breakpoints in TS, will come, if we will have a debugger, which is working.
